src/kotlin/flwr/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Move this into the global gitignore

Reply:
This has been moved in add-android-sdk, I'll pull the branch here.